control-plane/grpc/src/operations/volume/server.rs

```diff
@@ -0,0 +1,174 @@
+use crate::{
+    operations::volume::traits::VolumeOperations,
+    volume::{
+        create_volume_reply, get_volumes_reply, publish_volume_reply, set_volume_replica_reply,
+        share_volume_reply, unpublish_volume_reply,
+        volume_grpc_server::{VolumeGrpc, VolumeGrpcServer},
+        CreateVolumeReply, CreateVolumeRequest, DestroyVolumeReply, DestroyVolumeRequest,
+        GetVolumesReply, GetVolumesRequest, ProbeRequest, ProbeResponse, PublishVolumeReply,
+        PublishVolumeRequest, SetVolumeReplicaReply, SetVolumeReplicaRequest, ShareVolumeReply,
+        ShareVolumeRequest, UnpublishVolumeReply, UnpublishVolumeRequest, UnshareVolumeReply,
+        UnshareVolumeRequest,
+    },
+};
+use common_lib::types::v0::message_bus::Filter;
+use std::{convert::TryFrom, sync::Arc};
+use tonic::Response;
+
+/// RPC Volume Server
+#[derive(Clone)]
+pub struct VolumeServer {
+    /// Service which executes the operations.
+    service: Arc<dyn VolumeOperations>,
+}
+
+impl VolumeServer {
+    /// returns a new volumeserver with the service implementing Volume operations
+    pub fn new(service: Arc<dyn VolumeOperations>) -> Self {
+        Self { service }
+    }
+    /// coverts the volumeserver to its corresponding grpc server type
+    pub fn into_grpc_server(self) -> VolumeGrpcServer<VolumeServer> {
+        VolumeGrpcServer::new(self)
+    }
+}
+
+impl Drop for VolumeServer {
+    fn drop(&mut self) {
+        println!("Dropping volume server")
```

Comment:
Don't think this should be checked in..

Reply:
Yeah, added that for testing. Removing it.